src/Wt/WJavaScriptSlot.C

```diff
@@ -11,7 +11,7 @@
 
 namespace Wt {
 
-int JSlot::nextFid_ = 0;
+std::atomic<unsigned> JSlot::nextFid_ = 0;
```

Comment:
This should be non-atomic depending on `WT_THREADED`. Same comment for all the other ones.